RELEASE-NOTES.txt

```diff
@@ -10,6 +10,7 @@
 * [*] Block Editor: "Set as featured" button within image block settings. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3116]
 * [***] Block Editor: Audio block now available on WP.com sites on the free plan. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3523]
 * [*] Block Editor: Improve unsupported block message for reusable block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3621]
+* [**] Fix incorrect block insertion point after blurring the post title field. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3640]
```

Comment:
I see the rest of the entry items related to Gutenberg have the prefix `Block Editor:`, I'm not sure if it's a convention but we might want to include it, wdyt?

Reply:
The release notes for 17.6 have already been sent to Editorial on first week of the Sprint, and already sent to our translation vendor last week too, so those won't make it to the AppStore anyway.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/14892/commits/e24729b57deac65dab8aae1b6af420b62716b36b.